cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -183,11 +183,11 @@ def find_subnets(vpc_conn, vpc_id, identified_subnets):
             filters={'vpc_id': vpc_id, 'tag:Name': subnet_names})
 
         for name in subnet_names:
-            matching = [s.tags.get('Name') == name for s in subnets_by_name]
-            if len(matching) == 0:
+            matching_count = len([1 for s in subnets_by_name if s.tags.get('Name') == name])
```

Comment:
Sorry for missing this review! This change looks good; the original typo looks like it actually just meant to use a condition on the generator:
```py
matching = [s for s in subnets_by_name if s.tags.get('Name') == name]
```